README.md

```diff
@@ -36,56 +36,68 @@ concept Person identified by name  {
 🔎 Introspect the model using a [powerful set of APIs](https://docs.accordproject.org/docs/model-api.html)
 
 🎛 Convert the model to other formats:
-- JSON Schema
-- XML Schema
-- OData CDSL
-- GraphQL Schema
-- Java Classes
-- Go Types
-- C# Classes
-- TypeScript Classes
-- Protobuf Messages
-- PlantUML Diagrams
-- Mermaid UML Diagrams
-- Markdown (with embedded Mermaid)
-- OpenAPI v3 specification document
-- Apache Avro
+
+-   JSON Schema
+-   XML Schema
+-   OData CDSL
+-   GraphQL Schema
+-   Java Classes
+-   Go Types
+-   C# Classes
+-   TypeScript Classes
+-   Protobuf Messages
+-   PlantUML Diagrams
+-   Mermaid UML Diagrams
+-   Markdown (with embedded Mermaid)
+-   OpenAPI v3 specification document
+-   Apache Avro
 
 🕸 Publish your reusable models to any website, including the Accord Project [model repository](https://models.accordproject.org)
 
 Infer models from other formats:
-- JSON document
-- JSON Schema
-- OpenAPI v3 specification document
+
+-   JSON document
+-   JSON Schema
+-   OpenAPI v3 specification document
+
+## Features
+
+-   **Expressive Model Language**: Capture business models succinctly.
+-   **Serialization**: Convert instances to JSON.
+-   **Deserialization and Validation**: Validate instances from JSON.
+-   **Model Introspection**: Utilize a powerful set of APIs.
+-   **Format Conversion**: Convert models to various formats such as JSON Schema, GraphQL Schema, Java Classes, and more.
+-   **Model Publication**: Publish reusable models to websites or repositories.
```

Comment:
Concerto package itself does not provide any feature to publish a model to any repository.